frontend/public/components/graphs/area.tsx

```diff
@@ -53,9 +58,20 @@ export const AreaChart: React.FC<AreaChartProps> = ({
   xAxis = true,
   yAxis = true,
   chartStatus,
+  byteDataType = '',
 }) => {
   const [containerRef, width] = useRefWidth();
-  const getLabel = ({ datum: { x, y } }) => `${humanize(y).string} at ${formatDate(x)}`;
+  let processedData = data;
+  let unit: string = '';
+
+  //If user has speicified a type for incoming data is byte related
+  if (byteDataType) {
+    const result = processFrame(data, byteDataType);
+    processedData = result.processedData;
+    unit = result.unit;
+  }
+  const getLabel = getLabelFunction(humanize, formatDate, byteDataType, unit);
```

Comment:
The humanize function will work correctly here if we provide `initialUnit` and/or `desiredUnit` args. It's probably not a good idea to just bypass the humanize prop. We should also probably use `React.useCallback`:

```suggestion
  const getLabel = React.useCallback(({ datum: { x, y } }) => `${humanize(y, unit).string} at ${formatDate(x)}`, [formatDate, humanize, unit]);
```